ts/Extensions/Drilldown/Drilldown.ts

```diff
@@ -396,7 +396,7 @@ class ChartAdditions {
                 { colorIndex: pick(point.colorIndex, oldSeries.colorIndex) } :
                 { color: point.color || oldSeries.color },
             levelNumber = oldSeries.options._levelNumber || 0,
-            pointIndex = oldSeries.points.indexOf(point);
+            pointIndex = oldSeries.data.indexOf(point);
```

Comment:
I do not think that is correct, should fix https://github.com/highcharts/highcharts/blob/158e0c0c2694a7e24fb0f49ec03458b2b5b1690e/ts/Extensions/Drilldown/Drilldown.ts#L460 instead to keep support for ranges.

Reply:
@bre1470 Would using point.options & point.index work? or are those in reference to something else than the point array?

Reply:
`Series.data` includes all series points while `Series.points` includes only series points that are supposed to be rendered for the current view / render loop.